src/monogame/include/csharp_enums.h

```diff
@@ -0,0 +1,2104 @@
+//

```

Comment:
So is `src` the top level folder decided for the native code?

What will be in this `src` folder?  Is it just the MonoGame native C++ dll?

Reply:
`src/monogame` is the top level folder for the native code.

We should eventually move our src related stuff into one folder and not keeping it all mixed up in the top level, but that can come later once the native platform is fully working.